### PR TITLE
FE-1688 fix(load3d): move animation controls into a menubar timeline row

### PR DESCRIPTION
**Pull request by jtydhr88**
## Summary
The absolute-positioned AnimationControls overlapped the Load3DMenuBar top bar whenever an animated model (e.g. FBX) was loaded. Replace it, inside the node, with an AnimationMenuStrip rendered as a flow-layout timeline row between the viewport and the bottom bar, using the menubar's shared styles and popover exclusivity.

Load3DMenuBar now lays out top bar / viewport slot / timeline row / bottom bar in normal flow and Load3DScene renders inside the slot, so the renderer, letterboxing and the bottom-left view helper are sized to the visible area instead of extending under the opaque bars.

## Screenshots (if applicable)
Before
image

After
image

### CODE REVIEW

File: src/components/load3d/Load3D.test.ts [hunk 1]
```diff
@@ -238,18 +235,20 @@ describe('Load3D', () => {
...
-      expect(screen.getByTestId('animation-controls')).toBeInTheDocument()
+      expect(screen.getByRole('button', { name: 'Play' })).toBeInTheDocument()
     })
 
-    it('hides AnimationControls when the animation list is empty', () => {
+    it('hides the animation strip when the animation list is empty', () => {
       renderLoad3D()
-      expect(screen.queryByTestId('animation-controls')).not.toBeInTheDocument()
+      expect(
+        screen.queryByTestId('animation-menu-strip')
```
Comment (dante01yoon) [inline src/components/load3d/Load3D.test.ts:250]: suggestion: This assertion uses `animation-menu-strip`, while the component renders `load3d-animation-strip`, so this test would still pass if the strip were accidentally rendered for an empty animation list. Could we query the actual test ID (or assert that the accessible Play button is absent) so this integration test exercises the behavior it describes?

**Review by dante01yoon — APPROVED** [review]
The timeline-row layout and animation-control wiring look sound, and the formatter boundary cases are covered. CI is green. I left one non-blocking test-accuracy suggestion.
